packages/components/src/components/calendar/calendar.scss

```diff
@@ -138,7 +138,7 @@
         color: var(--text-general);
         box-sizing: border-box;
 
-        &:hover:not(.dc-calendar__cell--disabled) {
+        &--hover {
```

Comment:
This is changing the color or disabled calendar dates, is it part of the requirements?

Reply:
@matin-binary, because of this mistake flickering background. in calendar.jsx file has logic for tracking 'mouse' and add class, but this class no defined and never used, instead of this deal somebody put  ': hover{ }'. Accordingly, this thing brokes logic. First of all painting cell of date of the day, then painted red dot, when you don’t mo
<img width="1352" alt="Screenshot 2021-10-04 at 19 27 45" src="https://user-images.githubusercontent.com/88426229/135973928-eec59835-b4aa-4d44-927d-87d2241dbe64.png">
ve your mouse and just stop on the cell on top of the mouse, start flickering the background.

Reply:
Not sure if that's a mistake as it is related to parent selector using a pseudo-classes `:hover:not`.
you can add a separate selector for `&--hover`

Reply:
@matin-binary, I can, but then will become the next bug with the label 'flickering background-color of a cell during hover mouse on the red dot'.

Reply:
ok sure, lets see what QA has in store for us :)

Reply:
left as it was